tests/components/automation/test_template.py

```diff
@@ -462,8 +462,8 @@ async def test_wait_template_with_trigger(hass, calls):
     await hass.async_block_till_done()
     hass.states.async_set("test.entity", "hello")
     await hass.async_block_till_done()
-    assert 1 == len(calls)
-    assert "template - test.entity - hello - world - None" == calls[0].data["some"]
+    assert len(calls) == 1
+    assert calls[0].data["some"] == "template - test.entity - hello - world - None"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5456697)